source/common/network/connection_impl.cc

```diff
@@ -551,10 +551,10 @@ ClientConnectionImpl::ClientConnectionImpl(
   }
 
   if (source_address != nullptr) {
-    const int rc = source_address->bind(fd());
-    if (rc < 0) {
+    Api::SysCallResult result = source_address->bind(fd());
```

Comment:
nit: result here and below can be const.

Reply:
Addressed in 42d0771.